modules/ting_field_search/ting_field_search.module

```diff
@@ -823,29 +995,33 @@ function ting_field_search_facet_save($facet) {
   $facet = array_intersect_key($facet, array_flip($facet_fields));
 
   // Using a merge query to ensure only one facet per profile.
-  return db_merge('ting_field_search_facet')
+  db_merge('ting_field_search_facet')
     ->key($keys)
     ->fields($facet)
     ->execute();
 }
 
 /**
  * Delete facet settings associated with a profile.
+ *
+ * @param string $pid
+ *   The id of the profile to delete.
+ * @param string $facet
+ *   Facet to to used as condition for the deletions.
  */
-function ting_field_search_facets_delete($pid, $facet = NULL) {
+function ting_field_search_facets_delete($pid, $facet = '') {
```

Comment:
Why change the default value here?

Reply:
As all facets are strings, so having NULL as default don't makes sens. The function documentation declare $facet a string.